mlrun/api/db/sqldb/models.py

```diff
@@ -81,8 +82,8 @@ class Label(Base, BaseModel):
         )
 
         id = Column(Integer, primary_key=True)
-        name = Column(String)
-        value = Column(String)
+        name = Column(String(255, collation=SQLCollationUtil.collation()))
```

Comment:
Please add some reasoning for the DB schema changes in the PR description, for example for this one it would be something like:
MySQL requires length limit on string columns so added 255 limit on all columns